projects/1.16/assets/inventory-pets/inventorypets/lang/zh_cn.json

```
@@ -1,592 +1,592 @@
+    "item.inventorypets.holiday_gift": "礼物",
+    "item.inventorypets.banana": "香蕉",
+    "item.inventorypets.solstice_helmet": "冬至头盔",
+    "item.inventorypets.solstice_chestplate": "冬至胸甲",
+    "item.inventorypets.solstice_leggings": "冬至护腿",
+    "item.inventorypets.solstice_boots": "冬至靴子",
+    "item.inventorypets.solstice_sword": "冬至剑",
+    "item.inventorypets.pet_christmas_tree": "[宠物]圣诞树",
+    "item.inventorypets.pet_menorah": "[宠物]九灯烛台",
+    "item.inventorypets.pet_mishumaa_saba": "[宠物]七灯烛台",
+    "item.inventorypets.pet_politically_correct": "[宠物]节日快乐",
+    "item.inventorypets.petachieve_1": "宠物成就图标",
+    "item.inventorypets.petachieve_5": "宠物成就图标",
+    "item.inventorypets.petachieve_10": "宠物成就图标",
+    "item.inventorypets.petachieve_20": "宠物成就图标",
+    "item.inventorypets.petachieve_30": "宠物成就图标",
+    "item.inventorypets.petachieve_40": "宠物成就图标",
+    "item.inventorypets.petachieve_50": "宠物成就图标",
+    "item.inventorypets.petachieve_60": "宠物成就图标",
+    "item.inventorypets.petachieve_all": "宠物成就图标",
+    "item.inventorypets.petachieve_general": "宠物成就图标",
+    "item.gift.notholiday": "节日宠物只在十二月十四日到二十五日有效",
+    "item.gift.taken": "你今天已经拿到礼物了",
+    "item.gift.dates": "礼物只能在十二月十四日到二十五日使用",
+    "pet.holiday.dates": "请在十二月十四日到二十五日尝试",
+    "item.inventorypets.nugget_diamond": "钻石粒",
+    "item.inventorypets.nugget_obsidian": "黑曜石粒",
+    "item.inventorypets.nugget_coal": "煤粒",
+    "item.inventorypets.nugget_emerald": "绿宝石粒",
+    "item.inventorypets.nugget_iron": "铁粒",
+    "item.inventorypets.nugget_ender": "末影粒",
+    "item.inventorypets.nugget_lapis": "青金石粒",
+    "item.inventorypets.feed_bag": "饲料袋",
+    "item.inventorypets.feed_bag_new": "饲料袋",
+    "item.inventorypets.siamese_gift": "礼物",
+    "item.inventorypets.item_petrifier": "驯化器",
+    "item.inventorypets.patreon_head": "紫色的奶牛头",
+    "item.inventorypets.patreon_shirt": "帅气的T恤",
+    "block.inventorypets.cloud_block": "云朵方块",
+    "block.inventorypets.cloud_spawn": "云朵刷怪箱",
+    "block.inventorypets.sand_block": "沙子方块",
+    "block.inventorypets.sand_spawn": "沙子刷怪箱",
+    "block.inventorypets.stone_block": "石头方块",
+    "block.inventorypets.stone_spawn": "石块刷怪箱",
+    "block.inventorypets.space_spawn": "空间刷怪箱",
+    "block.inventorypets.nether_spawn": "下界刷怪箱",
+    "block.bed.nope": "你不能在这里睡觉",
+    "block.bed.allPlayersNotSleeping": "所有玩家都必须在床上",
+    "entity.inventorypets.mini_quantum_blaze_entity": "迷你量子烈焰人",
+    "entity.inventorypets.mini_quantum_enderman_entity": "迷你量子末影人",
+    "entity.inventorypets.bill_gates_entity": "比尔·盖茨",
+    "entity.inventorypets.steve_ballmer_entity": "史蒂夫·鲍尔默",
+    "entity.inventorypets.satya_nadella_entity": "萨提亚·纳德拉",
+    "entity.inventorypets.anvil_pet_entity": "活体铁砧",
+    "entity.inventorypets.bed_pet_entity": "活体床",
+    "entity.inventorypets.banana_entity": "香蕉",
+    "entity.inventorypets.apple_entity": "苹果",
+    "entity.inventorypets.golden_apple_entity": "金苹果",
+    "entity.inventorypets.siamese_entity": "暹罗猫头颅",
```

```suggestion
    "entity.inventorypets.siamese_entity": "暹罗猫的头",
```
按原版命名规律，非骷髅类生物对应的Head物品应使用“的头”而非“头颅”。